UploadDaemon/App.config

```diff
@@ -1,6 +1,24 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
 <configuration>
-    <startup> 
-        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.1" />
-    </startup>
+  <startup>
+    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.1" />
+  </startup>
+  <system.net>
+    <!-- By default, the upload daemon uses the IE proxy settings.
```

Comment:
As noted in the user guide, these are system-wide settings, not IE settings. In particular, they are also used by Edge, and I thing even by Chrome and Firefox if not specified otherwise.